lib/travis/build.rb

```diff
@@ -10,6 +10,11 @@ module Build
     BUILD_DIR = File.join(HOME_DIR, 'build')
 
     class << self
+      def version
+        @version ||= `git rev-parse HEAD 2>/dev/null || \\
+                        echo "${SOURCE_VERSION:-fafafaf}"`.strip
```

Comment:
`fafafaf` as an internal meaning only to travis <- i collect these special tribal knowledge tidbits of importance. worth adding someplace official-ish?